#!/usr/bin/env python3
PACKAGE = "crazyflie_online_tracker"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("hover_height", double_t, 0, "A double parameter",    .5, .2,   1.5)
gen.add("hover_yaw", double_t, 0, "A double parameter",    0,  -3.14,   3.14)

exit(gen.generate(PACKAGE, "hover_controller", "HoverController"))
